get-started/index.md

```diff
@@ -140,3 +140,26 @@ and paste the output into the `Token` input field.
 Now that you have installed and deployed Cryostat and know how to access its
 web client, continue on to [Guides]({% link guides/index.md %}) for
 guides through various common actions and workflows.
+
+## [Uninstalling Cryostat Operator](#uninstalling-cryostat-operator)
+In order to ensure that objects created by the operator and recordings created
+by Cryostat are properly removed, the Cryostat Operator must be installed
```

Comment:
"must be installed" -> "must remain installed" or "must still be installed", maybe? At a first read, "must be installed" sounds like I have to go install it (again? a different one than I already did?), rather than as intended as a precondition

Reply:
I think this would still work well with the steps lower where you say "please reinstall it", too